week-1/exercises/decimal.js

```diff
@@ -1,6 +1,14 @@
 
 const num = 56.5467;
 
+const wholeNumberPart = Math.round(num) - 2;
```

Comment:
To get the whole number part you can use `Math.trunc(num)`.  This is truncate and remove the decimal places